frontend/src/main/web/lib/components/glossary/ActionCell.jsx

```diff
@@ -71,13 +101,68 @@ var ActionCell = React.createClass({
 
       var updateButton,
         cancelButton,
-        comment = (
-          <Comment
-            className="ml1/4"
-            readOnly={!this.props.canUpdateEntry || !canUpdateComment || isSaving}
-            value={this.state.entry.transTerm.comment}
-            onUpdateCommentCallback={this._onUpdateComment}/>
+        readOnlyComment = !this.props.canUpdateEntry || !canUpdateComment || isSaving,
+        disableCommentUpdate = !this._hasCommentChanged(),
+        saveCommentButton;
+
+      if(this.state.savingComment) {
+        saveCommentButton = (
+          <Button kind='primary' size={-1} disabled={disableCommentUpdate}
+            onClick={this._onUpdateComment} loading>
+            Update Comment
+          </Button>
+        );
+      } else {
+        saveCommentButton = (
+          <Button kind='primary' size={-1} disabled={disableCommentUpdate}
+            onClick={this._onUpdateComment}>
+            Update Comment
+          </Button>
```

Comment:
Looks like `loading` is the only difference, should be able to just do `loading={this.state.savingComment}` I think (check docs though)